### ISSUE TITLE
Make sure auto update is working

### ISSUE THREAD

**Issue by mipstian** [mipstian/catch#103]
After the recent 1.9.3 update, judging by the number of people reporting that 1.9.2 wasn't working with new ShowRSS, I'm starting to think autoupdate might not be working. It could be that Catch is an always on app, and as such Sparkle's default check times (right after launching the app the second time) never happen for at least some users.

Better investigate.

**Comment by do1ng** [comment]
I had this exact problem. Had problems with new ShowRSS, and apparently auto update wasn't working. Installed it manually and it's working fine now.

**Comment by mipstian** [comment]
Found the root cause. For whatever reason, Sparkle uses the build number instead of the app version. So currently it would compare build number "503" with appcast version "1.9.3", and the build number always compares as greater, so Sparkle assumes it's an old version. Womp.

I'm gonna see if I can change the appcast so it internally uses build numbers, but still shows a user-readable version to users.

**Comment by mipstian** [comment]
Ok I was able to fix remotely by updating the appcast. This should bring everybody to the latest version.

Unfortunately, the update window says "Catch 502 is now available - you have 1.9.2" which makes no sense. So I added the actual version number in an h2 in the description. Not great, but better than having broken autoupdate.

Why Sparkle would use build numbers instead of versions is beyond me. I think it's a relic from when those two fields were called "short version string" and "long version string". Regardless, I prefer using the build number field for build numbers even if it causes a weird string in the update window.
